src/components/Signup.vue

```diff
@@ -48,11 +48,30 @@
             {{ item }}
           </el-tag>
         </el-form-item>
+        <el-form-item
+          v-if="this.$route.fullPath.includes('#url')"
```

Comment:
Cool feature. I think we should put back the button to open this "prompt" by the way, otherwise you can't just set registration IP.